### ISSUE TITLE
[Info]: Requesting info on dragon fight mechanics

### ISSUE THREAD

**Issue by Saereth** [someaddons/SavageEnderDragon#26]
### Describe the bug you're experiencing

This is not a bug so much as request for documentation. I'm trying to debug issues with the dragon fight which may have conflicting ai issues with champions but without actually know how the dragon is meant to behave in each various stages it's been difficult. 

What behavior modifications have been made to the dragon in each phase? 

I'm aware of the respawning crystals and mob spawns, the arc of lightning around it when it lands. Something is occasionally dropping people to 1 heart during the fight. They also go through period of doing no damage the dragon at all.

The CF page mentions an anti flying mechanic but flight seems to work fine around it? One player reported the dragon just straight up dying without doing any damage to it 

![image](https://user-images.githubusercontent.com/9200909/194774961-a4501211-f08b-4481-9f01-ed2c6261482f.png)
![image](https://user-images.githubusercontent.com/9200909/194774975-fc6a7615-4f09-4c5a-9354-b7bfed2cbbbe.png)

Does this mod change the dragon's armor or damage resistance along with the HP? How is the scaling calculated for additional group members and how is the range of other players being counted determined. 

Basically something isnt right with this fight, there are 250 mods in the FTB Inferno pack though so there is a good chance it could be another mod's fault but without information on the way the modified behavior should work its been difficult to troubleshoot. Any addition info would be highly appreciated. 

### Reproducability

Just summon and fight the ender dragon in the FTB Inferno pack 

### Mod up to date

- [X] Before submitting this issue I updated to the newest version and reproduced it

### Minecraft version

1.18

### Modloader version

Forge

### Logs

https://drive.google.com/file/d/1imyc0KhsZq63uFojlNYcjWmdjPXGog3B/view?usp=sharing

**Comment by someaddons** [comment]
I've listed the mechanics on curseforge, if you want to know more just check it out ingame without other mods
